soccer/planning/Path.cpp

```diff
@@ -0,0 +1,36 @@
+#include "Path.hpp"
+#include <protobuf/LogFrame.pb.h>
+
+namespace Planning {
+
+// This method is a default implementation of draw() that works by evaluating
+// the path at fixed time intervals form t = 0 to t = duration.
+void Path::draw(SystemState* const state, const QColor& color,
+                const QString& layer) const {
+    Packet::DebugRobotPath* dbg = state->logFrame->add_debug_robot_paths();
+    dbg->set_layer(state->findDebugLayer(layer));
+
+    // Get the closest step size to a desired value that is divisible into the
+    // duration
+    const float duration = getDuration();
+    const float desiredStep =
+        0.25;  // draw the path by interpolating every x seconds
+    const float segmentCount = ceilf(duration / desiredStep);
+    const float step = duration / segmentCount;
```

Comment:
I mean no matter which way we do that, its the same problem especially with the 0.25*numsegments and the ceilf. 
if duration = 0.250000000001 and the ceilf gets ceilf(1.00000000001), segment count = 2
Then it does t=0.25 and t=0.25000000000001 which would probs be the same in evaluate 
Either way floats are meh. Its just as likely either way.